app/registration/views/dashboard.html

```diff
@@ -2,7 +2,7 @@
 
 <visit-header patient="vm.patient"></visit-header>
 
-<check-in patient="vm.patient" on-check-in="vm.loadPatient();"></check-in>
+<check-in patient="vm.patient" on-check-in="vm.reload();"></check-in>
```

Comment:
Checkin button is not showing for those not checked-in yet
![image](https://user-images.githubusercontent.com/3810265/38862954-9c2f0806-4236-11e8-8c68-61909acbefdd.png)

Reply:
It should appear for users with receptionist role.